### ISSUE TITLE
Make language selector more accessible

### ISSUE THREAD

**Issue by juhakujala** [City-of-Helsinki/kerrokantasi-ui#698]
Currently the language selector button doesn't have any labels so users with screen readers don't easily know what that button is and what it does.

Also the button is lacking styles when it gets focused.
